index.html

```diff
@@ -35,5 +35,127 @@ <h1>I’m Benson.<br>Glad to see you!</h1>
+                <div class="image1">
+                    <img src="images/Snapshoot Portfolio (2).png" alt="project snapshot2">
+                </div>
+                <div class="body1">
+                    <h2>Multi-Post Stories</h2>
+                    <ul class="flexy">
+                        <li class="canopy">CANOPY</li>
+                        <li class="backend">Back End Dev</li>
+                        <li class="year">2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="linkss">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javasript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </article>
+            <article class="grid-item1">
+                <div class="grid-item3 tonic">
+                    <div class="image1">
+                        <img src="images/Snapshoot Portfolio (3).png">
+                    </div>
+                    <div class="body1">
+                        <h2>Tonic</h2>
+                        <ul class="flexy">
+                            <li class="canopy">CANOPY</li>
+                            <li class="backend">Back End Dev</li>
+                            <li class="year">2015</li>
+                        </ul>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="linkss">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javasript</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+            </article>
+            <article class="grid-item1 stories">
+                <div class="image1">
+                    <img src="images/Snapshoot Portfolio (4).png" alt="project snapshot4">
+                </div>
+                <div class="body1">
+                    <h2>Multi-Post Stories</h2>
+                    <ul class="flexy">
+                        <li class="canopy">CANOPY</li>
+                        <li class="backend">Back End Dev</li>
+                        <li class="year">2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="linkss">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javasript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </article>
+        </section>
+        <section class="about">
+            <div class="main">
+                <h1>About Myself</h1>
```

Comment:
You did well with this project and kudos to you 😄 . Just a reminder, please you're to have only one `h1` tag in your project to enhance search engine optimization and accessibility, so kindly change it to a lower `hx` tag 👍

Reply:
- Changed `About Myself` heading to lower `h2` for cleaner code standards.